lib/index.js

```diff
@@ -224,6 +226,13 @@ MapboxGeocoder.prototype = {
     return this;
   },
 
+  _onPaste: function(e){
+    var value = (e.clipboardData || e.clipboardData).getData('text');
```

Comment:
This seems redundant `(e.clipboardData || e.clipboardData)`. Did you mean `(event.clipboardData || window.clipboardData)`? I believe `window.clipboardData` is the IE support though not considered safe/secure.

Reply:
Good catch, I think that is supposed to be looking for `window.clipboardData`, I'll update the PR.